README.md

```diff
@@ -57,6 +57,27 @@ myFont.onload = evt => doSomeFontThings(evt);
 myFont.src = `./fonts/SourceCodeVariable-Roman.otf.woff2`;
 ```
 
+You can also pass a base64 string. In that case, you'll have to pass the filename as well, so the font type can de deducted from the extension:
```

Comment:
```suggestion
You can also pass in a file directly, e.g. using the [HTML Drag and Drop API](https://developer.mozilla.org/en-US/docs/Web/API/HTML_Drag_and_Drop_API). In that case, you'll need to pass the original filename explicity, so that the type of font can be determined from the extension:
```